bower.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "transportapi.js",
-  "version": "0.0.7",
+  "version": "0.1.0",
   "main": "dist/scripts/transportapi.min.js",
   "description": "A proxy API to TAPI endpoints",
-  "license": "TODO",
+  "license": "",
```

Comment:
Worth thinking about what this refers to (the transportapi.js code? the data from transportAPI), and putting something in here.

I assume the license refers to the transportapi.js code, so the `CC BY` license might be most appropriate ( https://creativecommons.org/licenses/by/4.0/ ) @harry-wood ?